src/organizations/components/OrgProfileTab.tsx

```diff
@@ -86,64 +141,37 @@ class OrgProfileTab extends PureComponent<Props> {
               <div className="code-snippet" data-testid="code-snippet">
                 <div className="code-snippet--text">
                   <pre>
-                    <code>{this.props.me.id}</code>
```

Comment:
does this need a conditional? Asking bc we do it elsewhere and on initial render the selector may not have returned the "me" value yet?

Reply:
I don't think so since the `me` here has a default value of `''` for the `id` property